es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,65 @@
+//const { createElement, noop } = require('../lib');
+
+import createElement from '../lib/create-element'
+import noop from '../lib/noop'
```

Comment:
Here we can combine this into a single import by importing from the `lib/index.js` file:

```JAVASCRIPT
import { createElement, noop } from '../lib';
```